src/pages/index.js

```diff
@@ -57,7 +55,7 @@ export const FINALISABLE_PAGES = {
     finaliseText: 'finalise_supplier_invoice',
   },
   externalSupplierInvoice: {
-    checkForError: checkForExternalSupplierInvoiceError,
+    checkForError: checkForSupplierInvoiceError,
```

Comment:
* externalSupplierInvoice key not needed
* supplierInvoice.checkForError = checkForSupplierInvoiceError